analytics_data_api/v1/serializers.py

```diff
@@ -569,7 +570,7 @@ def get_enrollment_modes(self, obj):
         return obj.get('enrollment_modes', None)
 
     def get_programs(self, obj):
-        return obj.get('programs', None)
+        return list(obj.get('programs', set()))
```

Comment:
Why are we potentially `list()`-ing `set()` here? Seems like we could set the default to be `[]` for a simpler line that effectively does the same thing